src/browser-frontend/selectors/ui-pages-selectors.js

```diff
@@ -44,8 +44,16 @@ export function getPageTabState(state, pageId) {
   return getPageVisuals(state, pageId).get('tabState');
 }
 
+export function getOptionalClass(state, pageId) {
+  return getPageVisuals(state, pageId).get('optionalClass');
+}
+
 // UI page computed properties getters.
 
+export function getPageDisplayOwner(state, pageId) {
+  return !!PagesSelectors.getPageOwnerId(state, pageId);
```

Comment:
nope, I was just separating UI concerns and domain concerns. figured the domain would want the tab ID, and the UI would just want a true/false value